src/components/TopBar/TopBar.vue

```diff
@@ -34,18 +34,22 @@
 						:class="{'description__in-chat' : !isInCall }">
 						{{ statusMessage }}
 					</p>
-					<template v-if="conversation.description">
-						<p v-tooltip.bottom="{
-								content: renderedDescription,
-								delay: { show: 500, hide: 500 },
-								autoHide: false,
-								html: true,
-							}"
-							class="description"
-							:class="{'description__in-chat' : !isInCall }">
-							{{ conversation.description }}
-						</p>
-					</template>
+					<NcPopover v-if="conversation.description"
```

Comment:
Just noticed, that popover might be wider than a description:
![image](https://github.com/user-attachments/assets/d64761a6-b0df-4b89-b6e1-423f2e27d1da)